.github/workflows/tag_image_push.yml

```diff
@@ -53,6 +62,17 @@ jobs:
               exit 1
           fi
 
+      - name: Check if it is latest-lts release
+        if: github.event_name != 'workflow_dispatch'
+        run: |
+          IS_LATEST_LTS_FILE=.github/latest_lts
```

Comment:
I see this code section is duplicated for RHEL
Anyway we can put this in .sh function?
then call like this:

`echo "IS_LATEST_LTS=$(check_is_lts_file )" >> $GITHUB_ENV`

Reply:
I need to rework this, so I will postpone addressing this for now

Reply:
understood